fetch.bs

```diff
@@ -6455,8 +6495,13 @@ method steps are to <a for=Headers>append</a> (<var>name</var>, <var>value</var>
  <li><p>If <a>this</a>'s <a for=Headers>guard</a> is "<code>immutable</code>", then <a>throw</a> a
  {{TypeError}}.
 
- <li><p>Otherwise, if <a>this</a>'s <a for=Headers>guard</a> is "<code>request</code>" and
- <var>name</var> is a <a>forbidden header name</a>, return.
+ <li>
+  <p>Otherwise, if <a>this</a>'s <a for=Headers>guard</a> is "<code>request</code>" and
+  (<var>name</var>, ``) is a <a>forbidden request-header</a>, return.
+
+  <p class=note>Passing a dummy <a>header value</a> to <a>forbidden request-header</a> ought not to
+  have any negative repercussions.
+  <!-- XXX Under what circumstances can this return statement be reached? -->
```

Comment:
Unless someone knows I'll file a follow-up issue on this and link that instead. It seems like this could be an assert.